changelog.txt

```diff
@@ -0,0 +1,5 @@
+TBA
+- Support Scrobbling on MacOS Catalina.
+
+2.1.37
+- Start of changelog.
```

Comment:
We don't need this. We have a changelog(s) already. See https://cdn.last.fm/client/Mac/updates.xml

Reply:
ah!

Reply:
Reverted